examples/pytorch/Makefile

```diff
@@ -0,0 +1,27 @@
+# The path to the image-classifier executable.
+LOADER?=~/src/build/glow/bin/image-classifier

```

Comment:
I believe these were taken from the example bundles Makefiles. They are just the default values in case an env var with the same name is not set -- this is what `?=` does. (Same on `GLOW_SRC` below too)

Reply:
@artemrakhov @jfix71 Yes, I just copy the env var from the example bundles Makefiles.

And I have tested the command in my server:

```bash
qijun@junq-5520:/mnt/c/Users/junq/works/glow/examples/pytorch$ LOADER=/mnt/c/Users/junq/works/glow/build/bin/image-classifier GLOW_SRC=/mnt/c/Users/junq/works/glow make run
python export_onnx_model.py --input_name data --model_name resnet18 --model_path resnet18.onnx
/mnt/c/Users/junq/works/glow/build/bin/image-classifier /mnt/c/Users/junq/works/glow/tests/images/imagenet/cat_285.png -image_mode=0to1 -m resnet18.onnx -model_input_name=data -cpu
Model: resnet18.onnx
 File: /mnt/c/Users/junq/works/glow/tests/images/imagenet/cat_285.png   Label-K1: 285 (probability: 16.4305)
```

The result is good